libraries/stdlib/samples/test/samples/collections/collections.kt

```diff
@@ -393,6 +393,12 @@ class Collections {
             assertPrints(chars.joinToString(limit = 5, truncated = "...!") { it.toUpperCase().toString() }, "A, B, C, D, E, ...!")
         }
 
+        @Sample
+        fun map() {
+            val numbers = listOf(75, 111, 116, 108, 105, 110)
+            assertPrints(numbers.map { it.toChar() }, "[K, o, t, l, i, n]")
```

Comment:
I'd consider using some more simple operation, for example squaring numbers or taking `length` property of string elements.

Reply:
Sure I will do that 👍